.github/scripts/test_trymerge.py

```diff
@@ -395,7 +397,7 @@ def test_get_checkruns_many_runs(self, *args: Any) -> None:
 
     def test_cancelled_gets_ignored(self, *args: Any) -> None:
         """Tests that cancelled workflow does not override existing successfull status"""
-        pr = GitHubPR("pytorch", "pytorch", 82169)
+        pr = GitHubPR("pytorch", "pytorch", 110367)
```

Comment:
Unrelated.  This is an old example what GitHub has cleaned up all check runs and the example needs to be updated.